onyo/lib/tests/test_commands_mkdir.py

```diff
@@ -18,13 +17,6 @@ def test_onyo_mkdir_errors(inventory: Inventory) -> None:
                   dirs=[dir_path],
                   message="some subject\n\nAnd a body")
 
-    # mkdir on existing asset path
```

Comment:
I would additionally add here a test that verifies onyo errors correctly when calling `onyo mkdir` on a directory which is already an asset directory :)

Related to #565